logger/scenario_logger/src/logger.cpp

```diff
@@ -87,50 +89,50 @@ void Logger::append(int level,
                     const std::string& description,
                     const std::string& from)
 {
-  scenario_logger_msgs::Log log;
+  scenario_logger_msgs::msg::Log log;
 
-  log.elapsed_time = ros::Time::now() - begin();
+  log.elapsed_time = rclcpp::Clock(RCL_ROS_TIME).now() - begin();
   log.level.level = level;
   log.categories = categories;
   log.description = description;
-  log.from = from;
+  log.source = from;
 
   append(log);
 }
 
 std::size_t Logger::getNumberOfLog() const
 {
   return data_.log.size();
-};
+}
 
 void Logger::updateMoveDistance(float move_distance)
 {
   data_.metadata.move_distance = move_distance;
 }
 
-boost::optional<boost::property_tree::ptree> toJson(const scenario_logger_msgs::Log& data)
+boost::optional<boost::property_tree::ptree> toJson(const scenario_logger_msgs::msg::Log& data)
```

Comment:
C++17 has `std::optional`, which would allows us to get rid of `boost::optional`. Given that all the distros that support ROS 2 Foxy have support for C++17, we could eventually switch to C++17.

For the sake of getting the ROS 2 port out of door, let's just continue using Boost, but I'll file a ticket about moving to C++17.

Reply:
Ticket filed in https://github.com/tier4/AutowareArchitectureProposal/issues/109